package.json

```diff
@@ -55,14 +55,15 @@
   },
   "dependencies": {
     "@oclif/command": "^1.0",
+    "@oclif/config": "^1.12.11",
     "@oclif/plugin-help": "^2.0",
-    "@stoplight/json-ref-resolver": "^1.2.2",
     "@stoplight/json": "^1.8.0",
+    "@stoplight/json-ref-resolver": "^1.2.2",
     "@stoplight/types": "^4.0.0",
     "@stoplight/yaml": "^2.1.0",
     "ajv": "6.x.x",
     "chalk": "^2.4.2",
-    "jsonpath": "https://github.com/stoplightio/jsonpath#78140eb1980d4ff385780e2fa12177735685ec3e",
+    "jsonpath": "https://github.com/stoplightio/jsonpath#dd5b17c801b9fe12c88f332d2434c0402d468a8c",
```

Comment:
Side note, should probably open PR in our jsonpath repo to get this commit merged into master. Was confusing to not see it there.

Reply:
True.